plugins/infrastructure/skills/memory-keeper/SKILL.md

```
@@ -0,0 +1,201 @@
+  sessions, or on demand when a domain has collected new issue/fix patterns.
+version: 1.0.0
+author: chuanseng-ng
+license: MIT
+allowed-tools: Read, Write, Bash
+---
+
+# Skill: Memory Keeper
+
+## Invocation
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)